src/components/FeeModal/FeeBreakdown.tsx

```diff
@@ -23,15 +21,25 @@ const AmountOrFree = ({ isFree, amountFiat }: { isFree: boolean; amountFiat: str
   )
 }
 
-export const FeeBreakdown = () => {
+type FeeBreakdownProps = {
+  feeModel: ParameterModel
+  inputAmountUsd: string | undefined
```

Comment:
Sanity checking this naming, is this named to generalize to trades and LP? In the context of trades this would normally be `quoteSellAmountUsdPrecision`.

Reply:
That's correct! The intent here is to generalize here